docs/usage-js.rst

```diff
@@ -33,71 +33,55 @@ Let’s have a look at an example configuration first. (Please also see the :ref
    {
      server: 'http://127.0.0.1:3000/api', // your URL to API endpoint the tests will run against
      options: {
-
-       'path': [],       // Required Array if Strings; filepaths to API description documents, can use glob wildcards
-
+       path: [],         // Required Array if Strings; filepaths to API description documents, can use glob wildcards
        'dry-run': false, // Boolean, do not run any real HTTP transaction
-       'names': false,   // Boolean, Print Transaction names and finish, similar to dry-run
-
-       'loglevel': 'warning', // String, logging level (debug, warning, error, silent)
-
-       'only': [],      // Array of Strings, run only transaction that match these names
-
-       'header': [],    // Array of Strings, these strings are then added as headers (key:value) to every transaction
-       'user': null,    // String, Basic Auth credentials in the form username:password
-
-       'hookfiles': [], // Array of Strings, filepaths to files containing hooks (can use glob wildcards)
-
-       'reporter': ['dot', 'html'], // Array of possible reporters, see folder lib/reporters
-
-       'output': [],     // Array of Strings, filepaths to files used for output of file-based reporters
-
+       names: false,     // Boolean, Print Transaction names and finish, similar to dry-run
+       loglevel: 'warning', // String, logging level (debug, warning, error, silent)
+       only: [],         // Array of Strings, run only transaction that match these names
+       header: [],       // Array of Strings, these strings are then added as headers (key:value) to every transaction
+       user: null,       // String, Basic Auth credentials in the form username:password
+       hookfiles: [],    // Array of Strings, filepaths to files containing hooks (can use glob wildcards)
+       reporter: ['dot', 'html'], // Array of possible reporters, see folder lib/reporters
+       output: [],       // Array of Strings, filepaths to files used for output of file-based reporters
        'inline-errors': false, // Boolean, If failures/errors are display immediately in Dredd run
-
-       'require': null, // String, When using nodejs hooks, require the given module before executing hooks
-
-       'color': true,
+       require: null,    // String, When using nodejs hooks, require the given module before executing hooks
+       color: true,
      },
+     emitter: new EventEmitter(), // listen to test progress, your own instance of EventEmitter
+     apiDescriptions: ['FORMAT: 1A\n# Sample API\n']
+   }
 
-     'emitter': EventEmitterInstance, // optional - listen to test progress, your own instance of EventEmitter
+.. js:data:: configuration
```

Comment:
Using the [JS Sphinx domain](http://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html#the-javascript-domain) here